Source/References/GameStartr-0.2.0.js

```diff
@@ -704,6 +705,17 @@ var GameStartr;
             GameStarter.ModAttacher.fireEvent("onGamePause");
         };
         /**
+         * Triggered Function for when the game is closed. The game is saved and the
+         * mod event is fired.
+         *
+         * @param GameStartr
+         */
+        GameStartr.prototype.onGameClose = function (GameStarter) {
+            // GameStarter.ItemsHolder.saveAll();
+            FullScreenPokemon.FullScreenPokemon.prototype.autoSave();
```

Comment:
You shouldn't call FSP log in the GameStartr base class. That defeats the purpose of having a generic base class.